### PR TITLE
this change seems sufficient, but not sure if there is a better approach

### PR DESCRIPTION
**Pull request by wdednam**
This change is [Reviewable](https://reviewable.io/reviews/highperformancecoder/minsky/231)

### CODE REVIEW

File: model/variable.cc [hunk 1]
```diff
@@ -503,6 +503,8 @@ void VariableBase::draw(cairo_t *cairo) const
     try
       {
         auto val=engExp();
+        
+        if ((sliderMax-sliderMin)/sliderStep > 1.0e04) sliderStep=(sliderMax-sliderMin)/1.0e04;   // ensure there are at most 10000 steps between sliderMin and Max. for ticket 1255.
```
Comment (highperformancecoder) [inline model/variable.cc:507]: Probably better to do this in VariableBase::adjustSliderBounds(), VariableBase::sliderSet() and initSliderBounds()

Comment (wdednam) [reply model/variable.cc:507]: Thank you. I wasn't sure. I wanted to add the fewest number of lines, and ended up settling for this after trying various options.

File: model/variable.cc [hunk 2]
```diff
@@ -502,7 +505,9 @@ void VariableBase::draw(cairo_t *cairo) const
   if (type()!=constant && !ioVar() && (vv.size()==1) )
     try
       {
-        auto val=engExp();
+        auto val=engExp();    
+        
+        double tmpSliderStep=(sliderMax-sliderMin)/sliderStep > 1.0e04 ? (sliderMax-sliderMin)/1.0e04 : sliderStep;
```
Comment (highperformancecoder) [inline model/variable.cc:510]: Did this prove to be necessary>

File: model/variable.cc [hunk 3]
```diff
@@ -423,6 +423,7 @@ void VariableBase::sliderSet(double x)
 {
   if (x<sliderMin) x=sliderMin;
   if (x>sliderMax) x=sliderMax;
+  if ((sliderMax-sliderMin)/sliderStep > 1.0e04) sliderStep=(sliderMax-sliderMin)/1.0e04;   // ensure there are at most 10000 steps between sliderMin and Max. for ticket 1255.
```
Comment (highperformancecoder) [inline model/variable.cc:426]: Probably worth extracting this calculation into a separate function/method, rather than repeating 3 times.

**Pull request merged**